apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -234,11 +234,17 @@ abort_test_contract(Config) ->
     %% Check that we get the abort string back.
 
     AbortString = <<"yogi bear">>,
-    {RevertValue,_} =
-        revert_call_compute_func(Node, APub, APriv, EncodedTestPub,
```

Comment:
I see this replaces the old test on outer revert with nested revert - rather than adding it. Is the outer revert from HTTP API tested in other place?

Reply:
I can add it back. It was a mistake in the old code, but it makes sense to test both entrypoints.

Reply:
I see addressed in 53dc639ea27e891bc2261d081826bf2303ae4720